src/components/common/_components/MobileSideBar.tsx

```diff
@@ -1,7 +1,7 @@
 import { MouseEvent } from 'react'
 import { Menu, X } from 'lucide-react'
 
-import GeneralSideBar from '@/app/_components/GeneralSidebar'
+import GeneralSideBar from '@/app/(chat)/_components/GeneralSidebar'
 
 interface MobileSideBarProps {
```

Comment:
Please move this interface to the types file and also make the structure like this:

Example:

types/
index.ts
chat.types.ts